common/api/src/main/java/com/dfsek/terra/api/util/ConstantRange.java

```diff
@@ -25,7 +25,7 @@ public ConstantRange(int min, int max) {
     }
     
     @Override
-    public Range multiply(int mult) {
+    public synchronized Range multiply(int mult) {
```

Comment:
There should never be a scenario where multiple threads are fighting over a `Range`. These changes should not be needed.

Eventually I'll just make ranges immutable anyway.